CMP/CmpWorkerService/App.config

```diff
@@ -46,7 +46,7 @@
     <add key="VMMTUSETTING" value="1438" />
     <add key="VMSPERVNETLIMIT" value="180" />
     <add key="EnforceAppAgAffinity" value="false" />
-    <add key="ImportAllAzureSubVms" value="false" />
+    <add key="ImportAllAzureSubVms" value="true" />
```

Comment:
Is this a setting that we need to take care of during installation time? (i.e., does it affect the wizard?) If not and we only need to put this in the config, then no prob.